src/smartstream/examples/Cargo.toml

```diff
@@ -1,6 +1,8 @@
 [workspace]
-members = ["test"]
-
+members = [
+    "filter",
+    "map",
+]
 
```

Comment:
Please make sure to run all `test_stream_fetch`

Reply:
Is this something that is set up already? Wouldn't we need to make new tests for the smart streams since they will edit the contents?

Reply:
I figured out what you meant, it's all fixed